test/fixtures/cookbooks/test/recipes/default.rb

```diff
@@ -23,11 +23,11 @@
 node.default['vagrant']['user'] = 'root'
 
 if platform_family?('debian')
-  execute 'current patches' do
-    command 'apt-get update'
-  end
-  package 'libvirt-dev'
-  node.default['vagrant']['plugins'] << { 'name' => 'vagrant-libvirt', 'env' => { CONFIGURE_ARGS: 'with-libvirt-include=/usr/include/libvirt with-libvirt-lib=/usr/lib' } }
+  apt_update
+  build_essential
```

Comment:
I think you may actually need to give the resource a name, i.e.:
```suggestion
  build_essential 'default'
```